rosidl_typesupport_c/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,123 @@
+This document is a declaration of software quality for the `rosidl_typesupport_c` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `rosidl_typesupport_c` Quality Declaration
+
+The package `rosidl_typesupport_c` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
```

Comment:
This is also talking about version 1.0, which is `1.ii`, so how about:

```suggestion
### Version Scheme [1.i]/[1.ii]
```

Reply:
Done